v2/node_read.go

```diff
@@ -31,11 +31,21 @@ func ReadJsonString(s string) (JsonNode, error) {
 	return unmarshal([]byte(s), json.Unmarshal)
 }
 
+// ReadJsonString reads a []byte as JSON and constructs a JsonNode.
+func ReadJsonBytes(s []byte) (JsonNode, error) {
+	return unmarshal(s, json.Unmarshal)
+}
+
 // ReadJsonString reads a string as YAML and constructs a JsonNode.
 func ReadYamlString(s string) (JsonNode, error) {
```

Comment:
```suggestion
// ReadYamlString reads a string as YAML and constructs a JsonNode.
func ReadYamlString(s string) (JsonNode, error) {
```

Please note, I'm aware, you didn't update this line. But there is a problem with it, so I'm reporting it